docs/Employer/7-Pay-Statement.md

```diff
@@ -50,7 +50,7 @@ Parameter | Type | Required | Description
             "individual_id": "be7b048c-a6f3-4194-a017-2f537d4f3565",
             "type": "regular_payroll",
             "payment_method": "check",
-            "total_hours": 80
+            "total_hours": 80.5,
```

Comment:
this will sound very dumb but 80.5 seems SO weird to me while 86.67 is very common. can we make it 86.67 to make it realistic? 🤣 🤣